JupyterNotebook_Sessao11-Questao4.ipynb

```diff
@@ -15,7 +15,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Questão 4\n",
+    "# Questão 4 blah blah +1\n",
```

Comment:
I don't understand this part, what does "blah" mean?

Reply:
I agree.